app/configuration.py

```diff
@@ -29,6 +30,8 @@
 
 from app.exceptions import ConfigurationFileNotFoundError, ConfigurationFileInvalidError
 
+ENV_VAR_KEYS = ("SITE_NAME", "SERIAL_NUMBER")
```

Comment:
can you explain why we need the allowlist now?

Reply:
asking just because I think we will need more keys available soon. The modbus settings also should be able to be configured with environment variables

Reply:
I was sticking with the initial strategy of accepting a limited whitelisted set of env vars. Do you think we should switch now to accepting any viable (`\W`) env var name? And if so should we check all config settings for env vars, rather than a limited subset?